main.go

```diff
@@ -416,9 +416,9 @@ func installOption(listAll bool, custBinPath, mirrorURL *string) {
 	tflist = lib.RemoveDuplicateVersions(tflist)
 
 	templates := promptui.SelectTemplates{
-		Active:   `> {{ .Version | green | bold }}`,
+		Active:   `»{{ .Version | green | bold }}`,
 		Inactive: `{{ .Version | cyan }}`,
-		Selected: `{{ "✔" | green | bold }} {{ .Version | cyan }}`,
+		Selected: `{{ "»" | green | bold }} {{ .Version | green }}`,
```

Comment:
Well, the `»` isn't exactly ASCII too. It is UTF-8. So basically `»` isn't any safer than `✔` 😿 
ASCII is 7-bit and 128 chars long table and hence has quite limited number of chars we can use for the selector. And this is why I suggested to have a switch to turn off using chars from the extended table (UTF-8) for dumb terminals along with not limiting users with UTF-compatible terminal.

What we can use as "safe" chars for selector is e.g. `> ` and for the selected item is e.g. `+` or `~`.

Let's revert this back to `>` and `✔` and deal with this later as this is not essential and is basically out of scope of this PR. Thanks.

Reply:
Can you link me to a proper ascii code list then? I pulled that from [here](https://theasciicode.com.ar/) so thought it would be safe. Worst case I'll go back to `>` as you suggested.

Reply:
Oh, my bad. Apologies, @Jukie. I missed the extended ASCII (the one up to 256 chars). Sorry for the noise. Just disregard my comment about `»`. Thanks.

Reply:
No worries, appreciate the concern!